src/sponsor/SponsorDialog.tsx

```diff
@@ -25,6 +28,30 @@ const SponsorDialog: React.VoidFunctionComponent = () => {
             icon="heart"
         >
             <div className={classNames(Classes.DIALOG_BODY, Classes.RUNNING_TEXT)}>
+                <h4>{i18n.translate('whoAreWe.heading')}</h4>
+                <p>
+                    {i18n.translate('whoAreWe.team')}&nbsp;
```

Comment:
Probably don't need a non-breaking space here. A regular space is fine.
```suggestion
                    {i18n.translate('whoAreWe.team')}{' '}
```

Reply:
Or if we ever support RTL languages, the link really should be filling in a placeholder in the translation.

Reply:
I went with `{' '}`.

If filling it in, I would have left out `Learn more` and just hyperlinked `team`, but couldn't get that to work right now. Seems OK for now though.

Reply:
for future reference, if you ever need to do inline styling, etc. in a translated string, you can use a placeholder like this: https://github.com/pybricks/pybricks-code/pull/1295/commits/ff777ef8352e3b63a7e4c6a4b3d664b2c689a656